### PR TITLE
[examples] Modern Data Stack + SDA Example

### PR DESCRIPTION
**Pull request by OwenKephart**

### CODE REVIEW

File: examples/modern_data_stack_assets/mds_dbt/config/.user.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+id: 4ba95205-8a33-462d-9e03-12d14cb82719
```
Comment (jmsanders) [inline examples/modern_data_stack_assets/mds_dbt/config/.user.yml:1]: I don't think we should check this in.

File: examples/modern_data_stack_assets/mds_dbt/models/users_augmented.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+select * from {{ source('postgres_replica', 'users') }}
```
Comment (jmsanders) [inline examples/modern_data_stack_assets/mds_dbt/models/users_augmented.sql:1]: Same - why the layer of indirection?

File: python_modules/dagster/dagster/grpc/compile.py [hunk 3]
```diff
@@ -104,8 +104,9 @@ def protoc(generated_dir: str):
 
     installed_pkgs = {
         # pylint: disable=not-an-iterable
-        pkg.key for pkg in pkg_resources.working_set
-    }  
+        pkg.key
```
Comment (jmsanders) [inline python_modules/dagster/dagster/grpc/compile.py:107]: Unrelated?

**Review by jmsanders — CHANGES_REQUESTED** [review]

File: examples/modern_data_stack_assets/README.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+This is an example of how to use the Software-Defined Asset APIs alongside Modern Data Stack tools
+(specifically, Airbyte and dbt).
```
[the remark this answers is not in the document]
Comment (OwenKephart) [reply examples/modern_data_stack_assets/README.md:2]: yep just want to get this merged in asap so the link is no longer broken -- plan on adding in the recipe-style instructions in here tmrw

File: examples/modern_data_stack_assets/mds_dbt/models/orders_cleaned.sql [hunk 5]
```diff
@@ -0,0 +1 @@
+select * from {{ source('postgres_replica', 'orders') }}
```
[the remark this answers is not in the document]
Comment (OwenKephart) [reply examples/modern_data_stack_assets/mds_dbt/models/orders_cleaned.sql:1]: this is just a demo dbt project, so I don't have these models actually do anything
